packages/naming.cell.match/cell-match.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const assert = require('assert');
+
+const BemCell = require('@bem/sdk.cell');
+const bemNamingParse = require('@bem/sdk.naming.entity.parse');
+const pathPatternParser = require('@bem/sdk.naming.cell.pattern-parser');
+
+const ALPHANUM_RE = '[a-z][\\w\\-]*';
+const resc = s => String(s).replace(/[\\^$*+?.()|[\]{}]/g, '\\$&');
+
+const buildPathParseMethod = ({ fs: { pattern, delims: dd = {}, scheme = 'nested' }, delims, wordPattern: wp = ALPHANUM_RE }) => {
+    const separation = pathPatternParser(pattern);
+
+    const dElem = 'elem' in dd ? dd.elem : (delims.elem || '__');
+    const dMod = 'mod' in dd ? dd.mod : (Object(delims.mod).name || (typeof delims.mod === 'string' && delims.mod) || '_');
```

Comment:
This code is very similar to [this](https://github.com/bem/bem-sdk/blob/9dbb34042f7b025c8ee427f9af81fd202d889b3f/packages/naming.cell.stringify/cell-stringify.js#L51-L52). Not right

Reply:
Need to move it to BemNamingConvention container too.

Fix planned in #304